q2_feature_classifier/classifier.py

```diff
@@ -133,20 +157,27 @@ def classify(reads: DNAIterator, classifier: Pipeline,
 
 
 _classify_parameters = {'chunk_size': Int, 'n_jobs': Int, 'pre_dispatch': Str,
-                        'confidence': Float}
+                        'confidence': Float, 'force_orientation':
+                        Str % Choices(['same', 'reverse-complement'])}
 
 
 plugin.methods.register_function(
-    function=classify,
+    function=classify_sklearn,
     inputs={'reads': FeatureData[Sequence],
             'classifier': TaxonomicClassifier},
     parameters=_classify_parameters,
     outputs=[('classification', FeatureData[Taxonomy])],
-    name='Classify reads by taxon.',
+    name='Pre-fitted sklearn-based taxonomy classifier',
     description='Classify reads by taxon using a fitted classifier.',
     parameter_descriptions={'confidence': 'Confidence threshold for limiting '
-                            'taxonomic depth. Negative value disables. '
-                            'Currently experimental. USE WITH CAUTION.'}
+                            'taxonomic depth. Negative value disables '
+                            'Currently experimental. USE WITH CAUTION',
+                            'force_orientation': 'Force assumption that reads '
+                            'have the same orientation as the reference '
+                            'sequences or that reads must be reversed and '
```

Comment:
I'm not exactly sure what you mean here, but I've changed the help text in accordance with my interpretation of your question. Please check the wording again (and feel free to change it if you would like to).